lib/plugins/package/lib/packageService.js

```diff
@@ -70,12 +71,21 @@ module.exports = {
     const functionObject = this.serverless.service.getFunction(functionName);
     const funcPackageConfig = functionObject.package || {};
 
+    // use the artifact in function config if provided
+    if (funcPackageConfig.artifact) {
+      const filePath = path.join(this.serverless.config.servicePath, funcPackageConfig.artifact);
+      functionObject.package.artifact = filePath;
+      return BbPromise.resolve(filePath);
+    }
+
     const exclude = this.getExcludes(funcPackageConfig.exclude);
     const include = this.getIncludes(funcPackageConfig.include);
     const zipFileName = `${functionName}.zip`;
 
     return this.zipService(exclude, include, zipFileName).then(artifactPath => {
-      functionObject.artifact = artifactPath;
```

Comment:
I found that function object returned by `serverless.service.getFunction()` has `artifact` inside `package`, similar to how we specify them in `serverless.yaml`,
```
functions:
  hello:
    handler: handler.hello
    package:
      artifact: artifactName.zip
```
Along with `exclude` and `include`. Is that not the case?